### DISCUSSION TITLE
Monitorare consumi Condizionatore Mitsubishi

### DISCUSSION THREAD

**Discussion by federoco** [indomus/forum#3149] [Hardware & Firmware]
Ciao a tutti,
sto domotizzando un condizionatore Mitsubishi, che ha due split dotati di inverter collegati ad un unico motore.
Sto seguendo la[vostra guida](https://indomus.it/progetti/domotizzare-un-condizionatore-tradizionale-con-broadlink-e-home-assistant-v2/) e adesso necessito di aggiungere il sensore per rilevare la quantità di assorbimento elettrico, come spiegato [qui](https://indomus.it/progetti/dedurre-lo-stato-di-un-elettrodomestico-non-domotico-su-home-assistant-con-sonoff-pow-o-similari/#sensori).

Mi sarei orientato sugli Shelly PM Mini Gen3 in quanto he ho già integrati diversi (2.5) per prese elettriche e avvolgibili e mi ci trovo molto bene.
Ho dei dubbi:
- confermate che è inutile optare per lo Shelly 1PM Mini Gen3, visto che non ho interruttori da collegare? L'unico motivo per sceglierlo è che sembra abbia la possibilità di mettere cavi anche più grandi di 1,5 mm nei connettori a vite
- è corretto che per poter dedurre lo stato di ciascuna delle due entità devo installare uno Shelly 1PM Mini Gen3 su ciascuno dei due split?
- per collere fisicamente lo shelly allo split mi basta ponticellare Fase, Neutro e messa a terra dello Shelly al cavo di alimentazione che arriva dal motore allo split?

Grazie

**Comment by indomus** [comment]
- corretto, puoi usare shelly 1pm mini gen3, se il calibro dei cavi te lo consente (nonché il carico massimo che deve sopportare).
- è corretto anche il secondo punto: un misuratore per split.
- sul terzo punto, devi far riferimento allo schema di connessione dello shelly in riferimento a tuo impianto. su questo ti consigliamo di consultare un elettricista.

**Comment by federoco** [comment]
Grazie, vi faccio sapere come va a finire.

**Reply by federoco** [reply]
In attesa di vedermi con l'amico elettricista sono arrivato a fine guida e vedo correttamente i due split in HA.
Avevo già verificato che il mio condizionatore è nell'elenco dei [codici disponibili](https://github.com/smartHomeHub/SmartIR/blob/master/docs/CLIMATE.md#available-codes-for-climate-devices) per SmartIR tuttavia la funzione "dry" non prevede l'impostazione della temperatura.
C'è modo di modificare il file .json in modo da rimuovere questa funzione?

In pratica vorrei eliminare temperatura impostata e possibilità di settaggio della stessa dall'entità di HA in modalità "dry" (le parti che ho evidenziato in rosso), come su telecomando.
(allego screenshot)
![dry](https://github.com/user-attachments/assets/6f1889bc-1f2e-4c4b-9563-4961ebd8e4d3)
![plancia dry](https://github.com/user-attachments/assets/2279ac18-a1a4-4d1e-aeb1-b0ea609a09d6)

**Reply by KC-inDomus** [reply]
no, non è possibile farlo.

**Comment by federoco** [comment]
L'elettricista è arrivato. Shelly installati. 
Li ho integrati in HA tramite la App nativa, come da voi suggerito. Adesso studio la vostra guida per dedurre lo stato dei condizionatori.
Visto che ho gli shelly 2.5 che controllano le tapparelle e alcune luci integrati tramite MQTT, ho pensato che per una questione di uniformità potrei integrare anche questi tramite App nativa.

- Ci sono controindicazioni?
- Nel caso, mi basta eliminare dal configuration.yaml la parte relativa a MQTT, fare reset degli Shelly e abbinarli tramite App?
- Le varie automazioni impostate sulle posizioni degli avvolgibili si conservano, giusto?

**Reply by KC-inDomus** [reply]
no non ce ne sono, anche se non ha troppo senso/utilità. puoi comunque abbinarli con l'app pur mantenendo la configurazione mqtt.
sì, le altre non hanno a che fare.
